src/components/shared/SearchCard/SearchCardAction/SearchCardAction.js

```diff
@@ -26,7 +25,7 @@ export const SearchCardAction = ({
       ? technology === rnaSeq
       : technology.find((x) => x === rnaSeq)
 
-  if (!pageRendered) return null
+  if (!experiment) return null
```

Comment:
I'm having trouble figuring out how this component could be rendered without being passed in an experiment?

Reply:
I’ve removed the `experiment` check from the `SearchCardAction` component, as it was a quick fix for this PR. The component now accepts the entire data structure (i.e., `experiment`) for GA4 events, and I've also updated the parent components to reflect this change (see at fc602f789c777c159f1254559c7784174ba19a49).

I'll update other `SearchCard`-related components to match this pattern in a separate issue (#363).

Additionally, I'll file a new issue to remove the usage of `usePageRendered` (which was added to prevent the hydration errors) as chatted in our 1:1.

Thank you, David!